frontend/src/canvas/CanvasContainer.js

```diff
@@ -183,6 +183,13 @@ const CanvasContainer = (props) => {
   });
 
   const pixelClicked = async (e) => {
```

Comment:
This solution seems to solve the issue because it clears the selected pixel super fast after clicking, so the user doesn't have much time to click again. But, I think it is still possible if the browser is slow / user is fast enough to cause double clicks.

I think using some kind of mutex will be needed to solve the issue since this function can be run asynchronously multiple times. But maybe there is some other crafty way of doing it.